src/test/python/mmlsparktest/vw/test_vw_cb.py

```diff
@@ -0,0 +1,88 @@
+# Prepare training and test data.
+import os
+import unittest
+import tempfile
+import pyspark
+
+from mmlsparktest.spark import *
+
+from mmlspark.vw.VowpalWabbitContextualBandit import VowpalWabbitContextualBandit
+from mmlspark.vw.VowpalWabbitFeaturizer import VowpalWabbitFeaturizer
+from mmlspark.vw.ColumnVectorSequencer import ColumnVectorSequencer
+
+from pyspark.sql.types import *
+from pyspark.ml import Pipeline
+
+class VowpalWabbitSpec(unittest.TestCase):
+
+    def get_data(self):
+        # create sample data
+        schema = StructType([StructField("shared_text", StringType()),
+            StructField("action1", StringType()),
+            StructField("action2_feat1", StringType()),
+            StructField("action2_feat2", StringType()),
+            StructField("chosenAction", IntegerType()),
+            StructField("label", DoubleType()),
+            StructField("probability", DoubleType())])
+
+        data = pyspark.sql.SparkSession.builder.getOrCreate().createDataFrame([
+            ("shared_f", "action1_f", "action2_f", "action2_f2=0", 1, 1.0, 0.8),
```

Comment:
action2_f2=0 is used as literal string, right? (e.g. passed directly to hashing)

Reply:
Yes